.github/workflows/doc_build.yml

```diff
@@ -48,6 +48,7 @@ jobs:
       run: |
           python3 --version
           sphinx-build --version
+          python3 -m pip install "breathe>=4.27"
```

Comment:
The above line updates the package to allow this PR to work. I'm not sure how to update the osgeo/proj container with the updated version of breathe, but would appreciate any pointers. An alternative idea is to merge the "Plots" CI job that uses conda-forge. It's not clear why there is a mixture of conda and containers used to check docs.

Reply:
The proj-docs source file is in docs/docbuild/Dockerfile . I've regenerated it locally and it got breathe 4.27 in it. I've checked it still runs fine with GDAL docs too since it is used by it. I'm pushing it now to Docker hub but it will take several hours

Reply:
> An alternative idea is to merge the "Plots" CI job that uses conda-forge. It's not clear why there is a mixture of conda and containers used to check docs.

Because I built the containers 5+ years ago, and I did the conda build recently. It would be reasonable to reconfigure the osgeo/proj-docs container to be conda-based so everything is the same baseline, but why break what's working (for PDAL, PROJ, MapServer, and GDAL too)

Reply:
We should maybe consider pushing the osgeo/proj-docs container to ghcr.io. There will be no bandwidth restrictions and we could build it with the GHA loop. I will investigate doing so.

Reply:
osgeo/proj-docs has now been refreshed. you can drop that change